spec/requests/api/merge_requests_spec.rb

```diff
@@ -268,6 +275,12 @@
       response.status.should == 200
       json_response['state'].should == 'closed'
     end
+
+    it "should return merge_request by iid" do
+      put api("/projects/#{project.id}/merge_request/#{merge_request.iid}", user), state_event: "close"
```

Comment:
Line is too long. [103/80]<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.